docs/generate.js

```
@@ -0,0 +1,331 @@
+      }
+      return result;
+    })();
+      
+    switch (this.type) {
+      case 'header':
+        this.type = `h${this.attrs.level}`;
+        break;
+        
+      case 'code_block':
+        // use code mirror to syntax highlight the code block
+        var code = this.content[0].text;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)